### DISCUSSION TITLE
Carddav Account keeps asking for password

### DISCUSSION THREAD

**Discussion by allinaseng** [Kozea/Radicale#1417] [Q&A general]
Running Radicale as a Carddav Server and after pushing it to clients phones, the account keeps asking for the password or there is some timeout happening that makes the contact list disappear?

Anyone know a way to combat this? 

Looking to keep the clients phones in sync with the server at all times.

**Comment by rngtng** [comment]
I’ve same problem on latest MacOS - did you found a solution?

**Comment by stan2100** [comment]
Hi rngtng and allinaseng. 
Have you found a solution to the problem yet? Maybe I have the same problem. Synchronisation works with all devices flawless, except with iPhones. On the iPhone, the password (for syncing) has to be re-entered from time to time.

**Comment by allinaseng** [comment]
Hello Stan2100 and rngtng, 
I was able to get CardDav to work on iPhones. I have the passwords set under two different accounts. For the general user, I enter their password for them / they enter it since its a generic password and it never prompts them again for the password. 
The way I was able to get it to work was: 

# Allow authenticated user to read all collections
[allow-everyone-read]
user: GeneralUser.*
collection: MasterAccount.*
permission: r

**Comment by quilime** [comment]
If you've changed any aspect of the auth strategy, try deleting the `.Radicale.cache` folder in the collection. It will regenerate with new sync-tokens for the new auth.

From the [manual](https://radicale.org/v3.html):

> Caches and sync-tokens are stored in the .Radicale.cache folder inside of collections. This folder may be created or modified, while the storage is locked for shared access. In theory, it should be safe to delete the folder. Caches will be recreated automatically and clients will be told that their sync-token isn't valid anymore.
